service/controller/machinepoolexp/handler/migration/testdata/old_azuremachinepool.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
```

Comment:
Test input, old experimental AzureMachinePool that will be migrated to the new non-experimental AzureMachinePool.